index.html

```diff
@@ -306,15 +305,18 @@ <h6>Social</h6>
           <a href="https://github.com/uyuni-project" target="_blank">
             <img src="img/github_white.svg" alt="Uyuni - Github" title="GitHub source repository">
           </a>
-          <a href="https://twitter.com/UyuniProject" target="_blank">
-            <img src="img/twitter.svg" alt="Uyuni - Twitter" title="Twitter">
+          <a href="https://l10n.opensuse.org/projects/uyuni/" target="_blank">
+            <img src="img/Weblate_logo.svg" width="24" height="24" alt="Uyuni - Weblate" title="Translate Uyuni to your language">
           </a>
           <a href="https://gitter.im/uyuni-project" target="_blank">
             <img src="img/gitter.svg" alt="Uyuni - Gitter" title="Gitter chat rooms">
           </a>
           <a href="https://www.youtube.com/channel/UCB0SkZFAw9vPCFeUIYqZQ5A" target="_blank">
             <img src="img/yt_icon_mono_dark.svg" width="24" height="24" alt="Uyuni - YouTube" title="YouTube channel">
           </a>
+          <a href="https://twitter.com/UyuniProject" target="_blank">
```

Comment:
Please place twitter below weblate, as it's on the other pages.

Otherwise move twitter to be last element at the other pages.